distributed/tests/test_worker.py

```diff
@@ -1019,7 +1019,10 @@ def f(x):
 @gen_cluster(client=True)
 async def test_get_client_coroutine(c, s, a, b):
     async def f():
-        client = await get_client()
+        # TODO: `await get_client()` raises a deprecation warning to use
+        # `async with get_client()` and that will kill the workers' client
+        # if you do that. We really don't want users to do that.
```

Comment:
this isn't true anymore - but we still don't want to `await get_client()`

```suggestion
        # TODO: the existence of `await get_client()` implies the possibility
        # of `async with get_client()` and that will kill the workers' client
        # if you do that. We really don't want users to do that.
        # https://github.com/dask/distributed/pull/6921/
```